### ISSUE TITLE
Custom thumb view.

### ISSUE THREAD

**Issue by vaukalak** [jeanregisser/react-native-slider#47]
Hey, and great project!

It would be nice to have a possibility, to setup a custom thumb View, for example a view wrapped in a tooltip https://github.com/chirag04/react-native-tooltip .

**Comment by vaukalak** [comment]
Implemented in https://github.com/jeanregisser/react-native-slider/pull/48

**Comment by lewisbarber** [comment]
This is just what I'm looking for in an app I'm building. Any chance of getting this merged soon?? Thanks!

**Comment by vaukalak** [comment]
@lewisbarber you can ref my GH as dependency for now `"react-native-slider": "git+https://github.com/vaukalak/react-native-slider.git#f9fd0b07d69f8cac77c8193038fb9a012dd7866e",`

**Comment by Nicomalacho** [comment]
Any advance on this new feature ?

**Comment by vaukalak** [comment]
@Nicomalacho last PR was closed on Jun 13. So probably author is not available at the moment.

**Comment by GregTheGreek** [comment]
I'm assuming this feature is not being implemented? has anyone found something that works? I would consider making a PR?

**Comment by vaukalak** [comment]
@GregTheGreek yeah, sorry for that, will fix my PR near to the and of this week.

**Comment by vaukalak** [comment]
@GregTheGreek Added new PR, which actually fixes some migrations to rect 16: https://github.com/jeanregisser/react-native-slider/pull/88 , asa this will be merged, will update my PR.
